src/app/public/modules/flyout/flyout-instance.ts

```diff
@@ -15,19 +15,53 @@ import {
   SkyFlyoutMessageType
 } from './types/flyout-message-type';
 
+/**
+ * The `SkyFlyoutService` launches flyouts and returns a `SkyFlyoutInstance` object to represent
```

Comment:
Given that this is the description for `SkyFlyoutInstance`, i don't think we need the first half of this. I think we can boil the description down to `Represents a single displayed flyout.`

Reply:
Done.